sysutils/htop/Portfile

```diff
@@ -38,6 +38,8 @@ pre-configure {
     system -W ${worksrcpath} "sh autogen.sh"
 }
 
+configure.args      --disable-unwind
```

Comment:
Maybe have `-append` here, so that nothing get overridden if added by `legacysupport`?

Reply:
`legacysupport-1.1.tcl` uses `port::register_callback legacysupport::add_legacysupport`, which runs after the Portfile and its variant blocks have been executed, so it should be fine to explicitly set `configure.args` here. It's the legacy support PortGroup that would have to use `-append` to not override things.

Reply:
Thanks for a clarification.